docs/web-apps/automated-testing/cypress.md

```diff
@@ -119,3 +119,9 @@ You can run `saucectl` locally via Docker ([Installation Requirements](https://d
 ## Cypress Plugin for Sauce Labs
 
 If you would prefer to stay in Cypress, try the new [Cypress Sauce Labs Plugin](https://github.com/saucelabs/sauce-cypress-plugin). Connect to your Sauce Labs account from within your Cypress project to configure and run your tests directly from Cypress.
+
+## Limitations
+
+:::caution Special Characters in Test Names
+We recommend sticking to plain English when it comes to naming tests. Your tests are not guaranteed to run or their artifacts be visible in our platform should they contain any special characters, like umlauts, accents etc.
```

Comment:
```suggestion
We recommend that you avoid the use of special characters when naming your tests. If your test names include special characters (* ë à & # ^), they are not guaranteed to run or their artifacts may not be visible in our platform.  
```

Reply:
Do the limitations exist for any special character, like "/ \ [ ] : | < > + = ; ? *" ?

Reply:
^Same suggestion applies to all files.

Reply:
> Do the limitations exist for all special characters, like "/ \ [ ] : | < > + = ; ? *" ?

Correct, though the exact limitations can also differ depending on whether the test is running on our mac vs. windows VMs. Since there are many possibilities and reasons as to which special character can fail, it's best for the user to simply avoid any special characters, since it's hard to enumerate all the use cases.